### ISSUE TITLE
add styling of new menuing system which replaced UIDropDownMenu framework

### ISSUE THREAD

**Issue by frozn** [frozn/TipTac#315]
### Is your feature request related to a problem? Please describe.

From: *miDnight_SC2*

Hey I've just recently started using TipTac and after prepatch I was wondering i I remember correctly but, didn't it also skin the dropdown menu from unitframes?  as in right clicking the player or target frame?

Source: https://legacy.curseforge.com/wow/addons/tiptac-reborn/?comment=523

**Comment by Paladin7788** [comment]
hey thanks for opening - and the awnser on curseforge -  I guessed that the new dropdown style caused that and I was mostly asking if I remember it correctly that it skinned those too as I've just recently used tiptac for  the first time :D

**Comment by frozn** [comment]
Yes, the old drop down menu before this expansion has been skinned by TipTac. For example it's the one I use on the button "Misc" in TipTac's option.

I played with skinning the new menu system a little bit yesterday, but the new menu mostly is encapsulated in private functions that cannot be manipulated from outside. The question is whether there is still a way to adjust the styling. I have to dig deeper into this...
